packages/drivers/file-driver/.eslintrc.js

```diff
@@ -10,5 +10,6 @@ module.exports = {
     ],
     "rules": {
         "@typescript-eslint/strict-boolean-expressions": "off",
+        "import/no-nodejs-modules": ["error", { "allow": ["fs"] }],
```

Comment:
What are the uses for `file-driver`? Alway run in a node environment? Is so, why is fs special? Otherwise we should just disable this rule for this package.